Modules/Registration/Metricsv4/wrapping/itkMetricv4Types.wrap

```diff
@@ -58,8 +58,8 @@ _add_image_filter_types("MSITIM" "itk::MeanSquaresImageToImageMetricv4")
 _add_image_filter_types("ITIM" "itk::ImageToImageMetricv4")
 
 # not sure why we are articially adding double, but this is consistent across the Metricv4 types
-UNIQUE(_scalar_plus_double "${WRAP_ITK_SCALAR};D")
-_cartesian(_real_pointsets "${_scalar_plus_double}" "${ITK_WRAP_IMAGE_DIMS}")
+UNIQUE(_real_plus_double "${WRAP_ITK_REAL};D")
+_cartesian(_real_pointsets "${_real_plus_double}" "${ITK_WRAP_IMAGE_DIMS}")
```

Comment:
So this change here essentially picks up `itk.F` as an additional type? I guess this should be fine; I remember an issue with instantiating `Optimizersv4` with `float` but if the dashboard is green all should be good.

Reply:
Yes, exactly. The broken example I referenced in ITKSphinxExamples uses an `itk.F` point set. To me floats seem like the most useful template option because 1) these metrics are typically used in the context of registration which requires higher precision than scalar types such as `signed short`, and 2) `itk.D` is not often wrapped and is not included in the ITK Python packages distributed via PyPI.